main.rb

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env ruby
+require 'pry'
```

Comment:
- [ ] Please remove this it gives an error while executing.

Reply:
Thanks